tools/ReleaseTool/PrepCommand.cs

```diff
@@ -0,0 +1,328 @@
+
+            using (var reader = new StreamReader(ChangeLogFilename))
+            {
+                while (!reader.EndOfStream)
+                {
+                    var line = reader.ReadLine();
+
+                    if (line.StartsWith("## ") && !hasReplacedUnreleased)
+                    {
+                        newChangeLog.AppendLine(line);
+                        newChangeLog.AppendLine(string.Empty);
+                        newChangeLog.AppendLine(string.Format(ChangeLogReleaseHeadingTemplate, options.Version, releaseDate));
+
+                        hasReplacedUnreleased = true;
+                        continue;
+                    }
+
+                    if (hasReplacedUnreleased && shouldUpdateGdkVersion && line.StartsWith("### Changed")) {
+                        newChangeLog.AppendLine(line);
+
+                        isInChangedSection = true;
+                        continue;
+                    }
+
+                    if (isInChangedSection && line.StartsWith("-"))
+                    {
+                        newChangeLog.AppendLine(line);
+
+                        isInChangedBulletPoints = true;
+                        continue;
+                    }
+
+                    if (isInChangedBulletPoints && !line.StartsWith("-"))
+                    {
+                        newChangeLog.AppendLine(string.Format(ChangeLogUpdateGdkTemplate, options.Version));
+                        newChangeLog.AppendLine(line);
+
+                        shouldUpdateGdkVersion = false;
+                        isInChangedBulletPoints = false;
+                        isInChangedSection = false;
+                        continue;
+                    }
+
+                    // If there is no existing "Changed" section
+                    if (shouldUpdateGdkVersion && line.StartsWith("## "))
+                    {
+                        newChangeLog.AppendLine("### Changed");
+                        newChangeLog.AppendLine(string.Empty);
+                        newChangeLog.AppendLine(string.Format(ChangeLogUpdateGdkTemplate, options.Version));
+                        newChangeLog.AppendLine(string.Empty);
+                        newChangeLog.AppendLine(line);
+
+                        shouldUpdateGdkVersion = false;
+                        continue;
+                    }
+
+                    newChangeLog.AppendLine(line);
+                }
+            }
+
+            using (var writer = new StreamWriter(ChangeLogFilename))
+            {
+                writer.Write(newChangeLog.ToString());
+            }
+
+            gitClient.StageFile(ChangeLogFilename, UriKind.Relative);
+        }
+
+        private void UpdatePackerConfig(GitClient gitClient)
+        {
+            if (!File.Exists(PackerConfigFile))
+            {
+                return;
+            }
+            
+            Console.WriteLine($"Updating {PackerConfigFile}...");
+
+            JObject jsonObject;
```

Comment:
Similar comment about using JSON models - in fact you can probably use the one that's already defined in the Packer csproj

Reply:
Yup agreed, I can definitely use that :P